### PR TITLE
travis: update components

### PR DESCRIPTION
**Pull request by valentijnscholten**
travis, update:
- k8s
- helm
- minikube

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@ services:
   - docker
 env:
   global:
-    - K8S_VERSION=v1.13.4
-    - MINIKUBE_VERSION=v0.35.0
-    - HELM_VERSION=v2.13.0
+    - K8S_VERSION=v1.18.0
```
Comment (madchap) [inline .travis.yml:7]: Maybe try with a least aggressive update? FYI, Google GKE stable is on 1.14'ish. Maybe 1.16.11?

File: .travis.yml [hunk 2]
```diff
@@ -4,9 +4,9 @@ services:
...
+    - MINIKUBE_VERSION=v1.11.0
```
Comment (madchap) [inline .travis.yml:8]: Same here.. there is sooo many versions in between, it's scary :fearful: 
1.0 was the next version after 0.35.0.

I can see in their changelog that 1.4.0 was the first release to support k8s 1.16.0 and that maybe v1.5.2 could be a good pick? Who knows. Wanna try?

**Comment by valentijnscholten** [comment]
I thought this was just a quick update, but apparently not. Maybe someone who actually knows k8s can take a look, or we just leave it as is.

File: .travis.yml (hunk 1 above)
Comment (valentijnscholten) [reply .travis.yml:7]: I think we need to be mooaaaaarrrr aggressive ;-)

File: .travis.yml (hunk 2 above)
Comment (valentijnscholten) [reply .travis.yml:8]: Let's go 1.12!!!!!

**Comment by valentijnscholten** [comment]
Superseeded by #2826

**Pull request closed without merging**